doc/news/_preparation_next_release.md

```diff
@@ -196,6 +196,10 @@ you up to date with the multi-language support provided by Elektra.
 - <<TODO>>
 - <<TODO>>
 
+## Packaging
```

Comment:
Please also write about the publishing of Docker images, maybe even as highlight.